### PR TITLE
Completed theme support

### PR DESCRIPTION
**Pull request by agnivade**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
-    "text": "green",
-    "command-background": "black",
-    "command-foreground": "red",
-    "command-token": "white"
-  },
   "themes": {
-    "bw": {
-      "name": "bold, underline",
+    "simple": {
+      "name": "bold,underline",
```
Comment (waldyrious) [inline config.json]: why was the space removed on this line?

Comment (waldyrious) [inline config.json]: (and on other similar lines)

File: test/config.spec.js [hunk 2]
```diff
@@ -16,11 +16,18 @@ describe('Config', function() {
 
   var CUSTOM_INVALID =
     '{' +
-    '  "colors": {' +
-    '    "text": "pretty" ' +
+    '"themes": {' +
+    '  "simple": {' +
+    '    "name": "bold,underline",' +
+    '    "description": "#876992",' +
```
Comment (waldyrious) [inline test/config.spec.js]: would it make sense to change the variables "name", "description" to "commandName", "mainDescription" on this PR, or would that require touching much more of the program?

File: config.json (hunk 1 above)
Comment (agnivade) [inline config.json]: because it didn't matter.

File: test/config.spec.js (hunk 2 above)
Comment (agnivade) [inline test/config.spec.js]: yup, makes sense. No troubles.

File: config.json (hunk 1 above)
Comment (waldyrious) [inline config.json]: If it makes no difference, then I'd suggest keeping the spaces, since it's more readable.

Comment (agnivade) [inline config.json]: sure

**Comment by waldyrious** [comment]
LGTM. Are the Travis failures related to #103?

**Comment by agnivade** [comment]
Nah .. it was actually the space thing in the `.tldrrc` file. It **did** matter after all. :stuck_out_tongue_closed_eyes:  I just squashed and force pushed into the branch.

**Comment by waldyrious** [comment]
Hahah nice :) My inner perfectionist just got a treat :P

I'll wait for others to comment, otherwise I'll merge in a day or two. Sounds good?

**Comment by agnivade** [comment]
No rush. I am in the process of updating the README too.

**Comment by agnivade** [comment]
Pushing it.

**Comment by waldyrious** [comment]
Yay! Can't wait to play around with this :D

**Comment by agnivade** [comment]
Go ahead, I just published to the registry.

**Comment by waldyrious** [comment]
Ok, cool. Shouldn't a release have been made [here on github](https://github.com/tldr-pages/tldr-node-client/releases) as well? That's where I looked to see if a release had been made.

In fact, isn't there a way to automatically keep both releases in sync? Maybe a bot to update npm when a release is made on github?

**Comment by agnivade** [comment]
You are right, they should be in sync. I have been manually making releases on github. Will see what can be done on it.

**Pull request merged**